openedx/features/idea/admin.py

```diff
@@ -3,4 +3,7 @@
 
 from django.contrib import admin
 
-# Register your models here.
+from .models import Idea
+
```

Comment:
can we add a modeladmin and add list fields to show when viewing multiple ideas in the admin panel?

Reply:
done